src/main/groovy/io/seqera/wave/configuration/ScanConfig.groovy

```diff
@@ -66,6 +69,9 @@ class ScanConfig {
     @Nullable
     private String severity
 
+    @Value('${wave.scan.retryAttempts:1}')
```

Comment:
```suggestion
    @Value('${wave.scan.retry-attempts:1}')
```

Reply:
Let's change to `retry-attempts` also the one for blob uploads

Reply:
blob has it already

https://github.com/seqeralabs/wave/blob/646b7c54d1318c1ed0d16512783698a01d26e61e/src/main/groovy/io/seqera/wave/configuration/BlobCacheConfig.groovy#L92

should i change it to `retry-attempts`?

Reply:
yeah, in some cases MN mess up with camelCase notation

Reply:
then should we change this, too?

https://github.com/seqeralabs/wave/blob/646b7c54d1318c1ed0d16512783698a01d26e61e/src/main/groovy/io/seqera/wave/configuration/BuildConfig.groovy#L83

Reply:
done